R/loo.R

```diff
@@ -192,10 +192,10 @@ loo.array <-
            ...,
            r_eff = NULL,
            save_psis = FALSE,
-           cores = getOption("mc.cores", 1)) {
-
+           cores = getOption("mc.cores", 1),
+           is_method = "PSIS") {
```

Comment:
make sure that you propagate `sis` and `tis` to the `loo` class. Right now all `loo` objects are of class psis_loo and loo with associated print method. Please also add proper print methods for `tis_loo` and `sis_loo`.

Reply:
Indeed! This should be fixed.

Reply:
Now fixed